docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -947,6 +946,13 @@ public class CustomLoadBalancerConfiguration {
 
 NOTE:: `HealthCheckServiceInstanceListSupplier` has its own caching mechanism based on Reactor Flux `replay()`, therefore, if it's being used, you may want to skip wrapping that supplier with `CachingServiceInstanceListSupplier`.
 
+[[spring-cloud-loadbalancer-hints]]
+=== Spring Cloud LoadBalancer Hints
+
+Spring Cloud LoadBalancer allows you to set hints that will be passed to the LoadBalancer within the `Request` object and that can later be used in `ReactiveLoadBalancer` implementations that can handle them.
+
```

Comment:
A hint is just a String, right?

Reply:
Yes, it's similar to how it's worked in the Ribbon-based implementation, but if you think it should be something else, we can discuss it here.

Reply:
Was just verifying, wasn't 100% sure from the docs

Reply:
ok. will add info to docs.